openai-servant/src/OpenAI/Resources.hs

```diff
@@ -382,7 +383,8 @@ $(deriveJSON (jsonOpts 3) ''ChatResponse)
 ------------------------
 
 data ImageResponseData = ImageResponseData
-  { irdUrl :: T.Text
+  { irdUrl :: Maybe T.Text
```

Comment:
minor nit; could be nice to have a custom `fromJSON` with a sum type here

Reply:
I'll try to take care of this next week :)

Reply:
Done!